README.md

```diff
@@ -134,7 +165,22 @@ Your query here
 7. The office with more and less employees.
 
 ```
-Your query here
+WITH number_of_employees_per_address AS
+  (SELECT offices.address,
+          count(*)
+   FROM offices
+   INNER JOIN employees ON offices.id = employees.office_id
+   GROUP BY offices.address)
+  (SELECT *
+   FROM number_of_employees_per_address
+   ORDER BY COUNT DESC
+   LIMIT 1)
+UNION ALL
+  (SELECT address,
+          COUNT
+   FROM number_of_employees_per_address
+   ORDER BY COUNT ASC
+   LIMIT 1);
```

Comment:
Very good solution. The `WITH` operator allows you to reutilze a common query between the two cases.